src/core/Akka.Tests.Performance/Dispatch/ThreadPoolDispatcherThroughputSpec.cs

```diff
@@ -1,28 +0,0 @@
-﻿//-----------------------------------------------------------------------
-// <copyright file="ThreadPoolDispatcherThroughputSpec.cs" company="Akka.NET Project">
-//     Copyright (C) 2009-2015 Lightbend Inc. <http://www.lightbend.com>
-//     Copyright (C) 2013-2015 Akka.NET project <https://github.com/akkadotnet/akka.net>
-// </copyright>
-//-----------------------------------------------------------------------
-
-using Akka.Configuration;
-using Akka.Dispatch;
-
-namespace Akka.Tests.Performance.Dispatch
-{
-    public class ThreadPoolDispatcherColdThroughputSpec : ColdDispatcherThroughputSpecBase
```

Comment:
I've removed it, because we have the same classes in DefaultDispatcherColdThroughputSpec.cs file